### PR TITLE
308: Hide cryptic errors

### LINKED ISSUE #308: Don't show cryptic error on StoreDetailsPage
Currently, if the user is not connected to the internet and opens a detail page of a store, he gets to see some fancy GraphQl error message.
Make it disappear like magic :mage_woman: :mage_man: 

Screenshot_20210316-003542

### PR DESCRIPTION
**Pull request by Schedulaar**
Closes #308 .

Sorry for the large PR, it's mostly boring stuff...

https://user-images.githubusercontent.com/7025199/128580638-8a582729-1218-4ddb-980f-5f8b284a94e0.mp4

https://user-images.githubusercontent.com/7025199/128580636-c84ed8dd-39b0-4061-ae9b-b32124ff18eb.mp4

### CODE REVIEW

File: frontend/lib/map/preview/accepting_store_preview_card.dart
Comment (maxammann) [inline frontend/lib/map/preview/accepting_store_preview_card.dart:10]: What does `/*?*/` mean?

File: frontend/lib/store_widgets/accepting_store_summary.dart
Comment (maxammann) [inline frontend/lib/store_widgets/accepting_store_summary.dart:79]: `double.infinity` trice as much as ∞!

**Review by maxammann — APPROVED** [review]
Sadly the flutter diffs are very hard to review. I'll just test this as soon as I get the app compiling again :D

File: frontend/lib/map/preview/accepting_store_preview_card.dart
Comment (Schedulaar) [reply frontend/lib/map/preview/accepting_store_preview_card.dart:10]: This is to mark an optional type (which is used once the null-safety migration is started)

**Review by elKei24 — APPROVED** [review]
LGTM. Did only a quick skim over the code (\approx 4min) and did not notice any issues.

**Comment by Schedulaar** [comment]
@maxammann should I wait for you to test it before I merge it?

**Comment by maxammann** [comment]
@Schedulaar lets merge and then test. I'm already doing some testing on android/ios

**Comment by Schedulaar** [comment]
Ok 🥳

**Pull request merged**
